tests/FSharp.Compiler.ComponentTests/EmittedIL/Literals.fs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.ComponentTests.EmittedIL
+
+open Xunit
+open FSharp.Test.Utilities.Compiler
+
+module ``Literals`` =
+
+    [<Fact>]
+    let ``Literal attribute generates literal static field``() =
+        FSharp """
+module LiteralValue
+
+[<Literal>]
+let x = 7
+
+[<EntryPoint>]
+let main _ =
+    0
+         """
+         |> compile
+         |> shouldSucceed
+         |> verifyIL ["""
+.field public static literal int32 x = int32(0x00000007)
```

Comment:
For small things like this it's probably fine, but for the larger IL verification tests we'll likely want to keep the baseline files and load them during test running. Could we add an example of that?

Reply:
It's not supported yet, here's the issue: #10000